air/src/options.rs

```diff
@@ -406,6 +427,55 @@ impl Default for PartitionOptions {
     }
 }
 
+// BATCHING OPTION
+// ================================================================================================
+
+/// Represents the type of batching, using randomness, used in the construction of the DEEP
+/// composition polynomial.
+///
+/// There are currently two types of batching supported:
+///
+/// 1. Linear, also called affine, where the resulting expression is a multivariate polynomial of
+///    total degree 1 in each of the random values.
+/// 2. Algebraic, also called parametric or curve batching, where the resulting expression is
+///    a univariate polynomial in one random value.
+///
+/// The main difference between the two types is that algebraic batching has low verifier randomness
+/// complexity and hence is light on the number of calls to the random oracle. However, this comes
+/// at the cost of a slight degradation in the soundness of the FRI protocol, on the order of
+/// log2(N - 1) where N is the number of code words being batched. Linear batching does not suffer
+/// from such a degradation but has linear verifier randomness complexity in the number of terms
+/// being batched.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[repr(u8)]
+pub enum BatchingMethod {
+    Linear = 0,
+    Algebraic = 1,
+}
+
+impl Serializable for BatchingMethod {
+    /// Serializes `self` and writes the resulting bytes into the `target`.
+    fn write_into<W: ByteWriter>(&self, target: &mut W) {
+        target.write_u8(*self as u8);
+    }
+}
+
+impl Deserializable for BatchingMethod {
+    /// Reads [BatchingType] from the specified `source` and returns the result.
+    ///
+    /// # Errors
+    /// Returns an error if the value does not correspond to a valid [BatchingType].
```

Comment:
We should use "[`BatchingMethod`]" in these comments.

Reply:
fixed